### ISSUE TITLE
Coding error in px:html-outline step (in html-to-dtbook)

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/pipeline#832]
The King Arthur Big Book of Bread, which we are using for FIDO testing is failing the Pipeline conversion. The title passes EPUBcheck and has accessibility issues with accessibility metadata, page breaks, and aria roles.

The log file does not make it clear where the problem is.

I placed the title, which is copyrighted in the DAISY shared projects/Pipeline folder.

@rorme@daisy.org

**Comment by bertfrees** [comment]
Hi George, I'm not sure if I can access this folder. Can't find it.

**Comment by GeorgeKerscher** [comment]
Does this dropbox link work?
https://www.dropbox.com/scl/fi/ur3kbz374dxflqyd18pc5/King_Arthur_Baking_Company_B_George_Kerscher.epub?rlkey=h5ihtwgzon41mhnoqfblzj7i8 <https://www.dropbox.com/scl/fi/ur3kbz374dxflqyd18pc5/King_Arthur_Baking_Company_B_George_Kerscher.epub?rlkey=h5ihtwgzon41mhnoqfblzj7i8&dl=0> &dl=0
 
Best
George

**Comment by bertfrees** [comment]
> Does this dropbox link work?

Yes, thanks!
